inc/saineighbor.h

```diff
@@ -50,6 +50,7 @@ typedef enum _sai_neighbor_entry_attr_t
 
     /**
      * @brief Destination MAC address for the neighbor
+     * Valid only when SAI_NEIGHBOR_ENTRY_ATTR_IS_LOCAL == true
```

Comment:
Why change "validonly" to "Valid only when"? This might break metadata check.

Reply:
"@validonly" (with the @) is used to enable the metadata check.  @kcudnik  brought this issue up as well but I mentioned to him that the metachecker fails since the validonly metacheck only works with "CREATE_ONLY" flags.

Kamil was OK keeping it as a comment.  I just made a minor change to cleanup the comment.